src/pages/PlayPage.js

```diff
@@ -35,11 +37,30 @@ export default function PlayPage({
             {isPlaying ? (
               duration && <Timer duration={duration} />
             ) : (
-              <PlayButton onClick={onPlay}>&gt;</PlayButton>
+              <PlayButton onClick={onPlay}>
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="100"
+                  height="100"
+                  viewBox="0 0 102 102.194"
+                >
+                  <g transform="translate(1 1.101)">
+                    <path
+                      d="M94.735,41.928,16.161,1.285C9.777-2.016,0,1.187,0,9.351V90.617c0,7.324,9.085,11.738,16.161,8.066L94.735,58.06c7.009-3.613,7.031-12.519,0-16.132Z"
+                      transform="translate(0 -0.002)"
+                      fill="none"
+                      stroke="#ed67bf"
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                      strokeWidth="2"
+                    />
+                  </g>
+                </svg>
```

Comment:
Is this the same `svg` you have in `/assets/play.svg`? If so, why are you not using that one? Also You should probably add an aria-label or you could add a `title` and `desc` to your inline svg. Have a look here: https://css-tricks.com/accessible-svgs/#2-inline-svg